integration-tests/sapling-transactions-contract-with-multiple-sapling-states.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { ContractAbstraction, ContractProvider, RpcReadAdapter, SaplingStateAbstraction } from '@taquito/taquito';
+import { CONFIGS } from './config';
+import { InMemorySpendingKey, SaplingToolkit } from '@taquito/sapling';
+import BigNumber from 'bignumber.js';
+import { saplingContractDoubleJProto } from './data/sapling_test_contracts';
+import { SaplingStateValue } from '@taquito/michelson-encoder';
+
+
+CONFIGS().forEach(({ lib, rpc, setup }) => {
+  const Tezos = lib;
+  let saplingContract: ContractAbstraction<ContractProvider>;
+  let saplingStateIdLeft: string;
+  let saplingStateIdRight: string;
+  let aliceInmemorySpendingKey: InMemorySpendingKey;
+  let alicePaymentAddress: string;
+  const memoSize = 8;
+
+  describe(`Test interaction with sapling contract having multiple sapling states using: ${rpc}`, () => {
```

Comment:
What do you think about a shorter describe block name? Something like 
'Sapling contract interaction - multiple sapling states'

Reply:
Yeah maybe we should have a discussion about that. I try to stick to the format Michael is using for changing all tests description.

Reply:
yeah that sounds good